browser/ui/views/side_panel/playlist/playlist_side_panel_coordinator.cc

```diff
@@ -33,8 +38,26 @@ void PlaylistSidePanelCoordinator::CreateAndRegisterEntry(
 }
 
 std::unique_ptr<views::View> PlaylistSidePanelCoordinator::CreateWebView() {
-  return std::make_unique<PlaylistSidePanelWebView>(&GetBrowser(),
-                                                    base::RepeatingClosure());
+  const bool should_create_contents_wrapper = !contents_wrapper_;
+  if (should_create_contents_wrapper) {
+    contents_wrapper_ =
+        std::make_unique<BubbleContentsWrapperT<playlist::PlaylistUI>>(
+            GURL(kPlaylistURL), GetBrowser().profile(), 0,
+            /*webui_resizes_host=*/false,
+            /*esc_closes_ui=*/false);
+    contents_wrapper_->ReloadWebContents();
+  }
+
+  auto web_view = std::make_unique<PlaylistSidePanelWebView>(
+      &GetBrowser(), base::RepeatingClosure(), contents_wrapper_.get());
```

Comment:
From the system resource optimiziing point of view, we may want to destroy `contents_wrapper_` when it doesn't play anything. Maybe it could be done with closing callback? WDYT?